pywhyllm/suggesters/tuebingen_model_suggester.py

```diff
@@ -110,37 +112,46 @@ def _build_description_program(self, variable, use_context=False, ask_reference=
                 answer within the tags, <description></description>, and each research paper within the tags <reference></reference>."""
+                query["user"] += cleandoc(user_prompt)
 
             else:
-                query["user"] += f"""
-                    Let's think step-by-step to make sure that we have a proper and clear description. Then provide your final 
-                    answer within the tags, <description></description>."""
+                query["user"] += "\n" # cleandoc removes leading whitespace, so need to add separately
+                user_prompt = f"""Let's think step-by-step to make sure that we have a proper and clear description. Then provide your final 
+                answer within the tags, <description></description>."""
+                query["user"] += cleandoc(user_prompt)
 
         else:
-            query["system"] = f"""You are a helpful assistant for writing concise and peer-reviewed descriptions. Your goal 
+            sys_prompt = f"""You are a helpful assistant for writing concise and peer-reviewed descriptions. Your goal 
             is to provide factual and succinct description of the given concept."""
-            query["user"] = f""" Describe the concept of {variable}.
-                    In one sentence, provide a factual and succinct description of {variable}"""
+            user_prompt = f"""Describe the concept of {variable}.
+                    In one sentence, provide a factual and succinct description of {variable}."""
+            query["system"] = cleandoc(sys_prompt)
+            query["user"] = cleandoc(user_prompt)
 
             if ask_reference:
-                query["user"] += f""""
-                        Then provide two research papers that support your description.
+                query["user"] += "\n" # cleandoc removes leading whitespace, so need to add separately
+                user_prompt = f"""Then provide two research papers that support your description.
                         Let's think step-by-step to make sure that we have a proper and clear description. Then provide 
                         your final answer within the tags, <description></description>, and each research paper within the 
                         tags <paper></paper>."""
+                query["user"] += cleandoc(user_prompt)
             else:
-                query["user"] += f"""
-                        Let's think step-by-step to make sure that we have a proper and clear description. Then provide 
+                query["user"] += "\n" # cleandoc removes leading whitespace, so need to add separately
+                user_prompt = f"""Let's think step-by-step to make sure that we have a proper and clear description. Then provide 
                         your final answer within the tags, <description></description>."""
+                query["user"] += cleandoc(user_prompt)
```

Comment:
consider query["user"] += "\n" + cleandoc(user_prompt) instead of adding with 2 separate lines?

Reply:
good point, fixed.